models/buffer_disk_test.go

```diff
@@ -95,15 +65,29 @@ func TestBuffer_TrackingDroppedFromOldWal(t *testing.T) {
 	// call manually so that we can properly use metric.ToBytes() without having initialized a buffer
 	registerGob()
 
+	// Prefill the WAL file
+	path := t.TempDir()
+	walfile, err := wal.Open(filepath.Join(path, "123"), nil)
+	require.NoError(t, err)
+	defer walfile.Close()
```

Comment:
This defer should be unneeded since its explicitly called later right?

Reply:
No I don't think it is unnecessary because the test can fail in between (with the `require` calls) and the file would be left open. So the double close should be fine to not leak anything...